values.yaml

```diff
@@ -334,11 +334,22 @@ controllerrole:
     rules:
     - apiGroups:
       - ""
+      - extensions
```

Comment:
IIUC this looks overly broad and will allow access to secrets which should not be required for this service.

Reply:
It to give this permission (see for example https://github.com/jenkins-x/jenkins-x-platform/blob/master/templates/committer-role.yaml) and thus needs to have it.

https://kubernetes.io/docs/reference/access-authn-authz/rbac/#privilege-escalation-prevention-and-bootstrapping